src/main/java/org/wildfly/security/util/Alphabet.java

```diff
@@ -269,4 +269,41 @@ public int decode(final int codePoint) {
             }
         };
     }
+    /**
+     * The alphabet used by PicketBox project base 64 encoding.
+     * {@code 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz./}
+     */
+    public static final Base64Alphabet PICKETBOX_COMPATIBILITY = new Base64Alphabet(false) {
```

Comment:
Wondering if this should be immediately deprecated to avoid anything else needing it.